src/borg/archive.py

```diff
@@ -21,10 +21,11 @@
 from .chunker import Chunker
 from .compress import Compressor
 from .constants import *  # NOQA
+from .crypto import IntegrityError as IntegrityErrorBase
```

Comment:
Do we still need IntegrityErrorBase if key.py wraps low level crypto IntegrityError anyway?